packages/wasm-parser/test/index.js

```diff
@@ -66,6 +65,11 @@ describe("Binary decoder", () => {
 
   compare(testSuites, getActual, getExpected);
 
+  function makeBuffer(...splitedBytes) {
+    const bytes = [].concat.apply([], splitedBytes);
+    return new Uint8Array(bytes).buffer;
+  }
```

Comment:
Dev deps are fine we can keep the helper here or there's another reason?

Reply:
Lerna will complain about it even when it is devdep